browser/js/user-dataset-details/user-dataset-details.js

```diff
@@ -0,0 +1,25 @@
+app.config(function ($stateProvider) {
```

Comment:
Are the one-dataset-details and user-dataset-details states different enough to need to be separate views or can they be combined?